flink-core/src/test/java/org/apache/flink/api/common/operators/base/FlatMapOperatorCollectionTest.java

```diff
@@ -42,6 +43,7 @@
 import java.util.List;
 import java.util.concurrent.Future;
 
+/** The test for flat map operator. */
```

Comment:
Such code-style clean-ups can be placed in a separated hotfix commit.

Reply:
These code-style clean-ups have been placed into the commit `[hotfix] Fix the lacking class comments for the tests using TaskInfo`.